### DISCUSSION TITLE
Am I stupid or is this basically unusable starting from scratch?

### DISCUSSION THREAD

**Discussion by Hawpe** [htpcBeginner/docker-traefik#239] [Support]
The README instructions are way too broad, the tutorial from the website is almost 2 years old and completely outdated and doesn't match this repo at all. Many variables are changed and not explained at all. It almost seems easier to start from scratch than to try and debug all the issues I'm seeing with this.

Don't mean to sound ungrateful as I realize there is a lot of work behind this but it seems like it's just someone's configuration that works specifically for them and you have to change everything to make it work for you, which is pretty far from the spirit of the original tutorial.

**Comment by SimpleHomelab** [comment]
I get it. Realize that this repo is to share what I do and inspire people to do it themselves. Although in many cases it can be a turnkey solution, most applications will require tweaking. 

I try my best to put comments and notes where I can. But updating all the documentation, repo, and guides is not an easy task. I do my best but rely on tinkerers also to self-learn - hence why it is hard to find a repo/guide from anyone that works for everything.

I know the guides are due an update. I am working on it, however, with Ubuntu 22.04 right around the corner, i would prefer to test and write the guide for it than having to revise it again.

**Comment by Hawpe** [comment]
I understand. Sorry if I sounded abrasive, I realize it's a lot of work. No entitlement here, just confusion/frustration as I don't see anyone else here mentioning the issues I've been experiencing.

I'll figure it out in the end but I think I'll likely start from scratch and look at your repo as supplemental help rather than as an all-in-one solution.

Thanks for all the work and the guides though!

**Reply by g0ldfngr** [reply]
I do recommend starting with Traefik and getting up and running and your certificate pulled,  then slowly add services one by one and testing along the way.  By attempting all of them at once and troubleshooting it will feel very overwhelming.

**Reply by Cartache** [reply]
I completely agree. It's a long process that requires patience and organisation. I have to say I went through it, it was sometimes difficult but in the end I have a working version that is regularly improved. 
Thanks to this worth full tutorial I am able to reinstall a complete server in a couple of hours (I have more than 50 services in my configuration). Previously it took me several days.

**Comment by Cartache** [comment]
> I get it. Realize that this repo is to share what I do and inspire people to do it themselves. Although in many cases it can be a turnkey solution, most applications will require tweaking. 
> 
> 
> 
> I try my best to put comments and notes where I can. But updating all the documentation, repo, and guides is not an easy task. I do my best but rely on tinkerers also to self-learn - hence why it is hard to find a repo/guide from anyone that works for everything.
> 
> 
> 
> I know the guides are due an update. I am working on it, however, with Ubuntu 22.04 right around the corner, i would prefer to test and write the guide for it than having to revise it again. 

Thanks for your efforts and your modesty. It's a great piece of work and the best I could find. It helped me a lot.
